openedx/core/djangoapps/content_tagging/helpers/objecttag_export_helpers.py

```diff
@@ -135,7 +135,7 @@ def _get_library_block_tagged_object(
 
 
 def build_object_tree_with_objecttags(
-    content_key: LibraryLocatorV2 | CourseKey,
+    content_key: LibraryLocatorV2 | CourseKey | LibraryCollectionKey,
```

Comment:
Is it ok to use [`content_tagging.types.ContentKey`](https://github.com/openedx/edx-platform/pull/35383/files#diff-f5e362bec266e15b388c6ee8522096c303eb09b94883e8689c0754b05c5a4b92R12) here?

Reply:
Updated: https://github.com/openedx/edx-platform/pull/35383/commits/4bc07a0570210dc607c62d91d0bbcd996b8db358